test/integrationtests/skills/skill_tester.py

```diff
@@ -327,6 +304,92 @@ def run(self, loader):
             s.get_response = orig_get_response
             s.settings = original_settings
 
+    def play_query(self, s, test_case):
```

Comment:
Would it make sense for these methods to be `emit_play_query` or `send_play_query`?

Reply:
It definitely would, I will update those names and some of the other

Reply:
Names updated!